src/NzbDrone.Core/Sonarr.Core.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="FluentMigrator.Runner" Version="3.3.2" />
     <PackageReference Include="FluentMigrator.Runner.SQLite" Version="3.3.2" />
     <PackageReference Include="FluentValidation" Version="8.6.2" />
-    <PackageReference Include="SixLabors.ImageSharp" Version="2.1.3" />
+    <PackageReference Include="SixLabors.ImageSharp" Version="3.0.1" />
```

Comment:
Changelog:
https://github.com/SixLabors/ImageSharp/releases

I tested with one banner, works good.

Also I noticed pretty interesting comment here:
https://github.com/Sonarr/Sonarr/blob/develop/src/NzbDrone.Core/MediaCover/MediaCoverService.cs#L42

Looks like we can remove this semaphore, but I don't have anything on arm to test. Should I create a pull request?

Reply:
That depends whether there is hardware acceleration under .net, without being able to test myself I don't think it's worth removing.

Reply:
@markus101, The comment says that there is no HA on mono. Okay, probably I will flash my raspberry pi and check some day :)

Reply:
Yeah, but I don't want to assume because it says mono that it only affected mono, we didn't have arm based Windows support then either.